src/types/homepage.ts

```diff
@@ -78,6 +78,19 @@ export interface AnnouncementsBlockSection {
   subtitle?: string
   announcement_items: AnnouncementOption[]
 }
+export interface TextcardsCard {
```

Comment:
suggest renaming to TextCardItem

Reply:
Changed in https://github.com/isomerpages/isomercms-frontend/pull/1505/commits/21b7ee1e6bc696c409c65b4d95b9638dcc458779!